app/src/main/java/com/sunasterisk/youtubebackground/data/source/remote/fetchjson/ParseJson.kt

```diff
@@ -13,50 +13,49 @@ class ParseJson {
         videos = videosParseJson(jsonObject.getJSONArray(PageEntry.ITEMS))
     )
 
-    fun  videosParseJson(jsonArray: JSONArray): MutableList<Video>? {
-        var listVideo = mutableListOf<Video>()
-        for (item in 0..jsonArray.length() - 1) {
-            listVideo.add(
-                Video(
-                    jsonArray.getJSONObject(item).getString(VideoEntry.ETAG),
-                    jsonArray.getJSONObject(item).getString(VideoEntry.ID),
-                    snippetParseJson(jsonArray.getJSONObject(item).getJSONObject(VideoEntry.SNIPPET)),
-                    contentDetailParseJson(jsonArray.getJSONObject(item).getJSONObject(VideoEntry.CONTENT_DETAILS)),
-                    statisticParseJson(jsonArray.getJSONObject(item).getJSONObject(VideoEntry.STATISTICS))
+        fun videosParseJson(jsonArray: JSONArray): MutableList<Video>? {
+            var videos = mutableListOf<Video>()
+            for (item in 0 until  jsonArray.length() - 1) {
```

Comment:
`until` và `..` có gì khác nhau 
https://kotlinlang.org/docs/tutorials/kotlin-for-py/loops.html#for

Reply:
until là 0 tới jsonArray.length - 1
còn .. là từ 0 tới jsonArray.length